includes/reports/overview/list-table/class-sensei-reports-overview-list-table-courses.php

```diff
@@ -63,19 +71,26 @@ public function get_columns() {
 			return $this->columns;
 		}
 
-		$total_completions      = Sensei_Utils::sensei_check_for_activity(
-			array(
-				'type'   => 'sensei_course_status',
-				'status' => 'complete',
-			)
-		);
-		$total_average_progress = $this->reports_overview_service_courses->get_total_average_progress();
+		$total_courses_ids       = $this->get_all_item_ids();
```

Comment:
This could be renamed to `$all_course_ids`.

Reply:
Renamed: https://github.com/Automattic/sensei/pull/5091/commits/ac85ff2ca1cf74cadcec0d40bc25f4256e0eb255